### PR TITLE
Document that the OpsWorks provider accepts a region option

### PR DESCRIPTION
**Pull request by billhorsman**
I'm not sure what the default region is but I think it's wise to specify it explicitly. This fixes issue #1302.

### CODE REVIEW

**Comment by billhorsman** [comment]
Sorry, I referenced the wrong issue in the commit message. This does in fact fix issue #1302

**Comment by plaindocs** [comment]
Thanks for this @billhorsman 

Looks like the default is set at https://github.com/travis-ci/dpl/blob/73eabc709312d91aba2886b99cea6c105c2f15b7/lib/dpl/provider/ops_works.rb#L15

**Comment by billhorsman** [comment]
Yes, you're right @plaindocs. I looked for that but couldn't find it. I'll update my changes to mention the default (and update from master again).

**Comment by billhorsman** [comment]
OK, ready again for your @plaindocs

File: user/deployment/opsworks.md [hunk 1]
```diff
@@ -29,7 +32,7 @@ You can also have the `travis` tool set up everything for you:
 travis setup opsworks
 ```
 
-Keep in mind that the above command has to run in your project directory, so it can modify the `.travis.yml` for you.
+Keep in mind that the above command has to run in your project directory, so it can modify the `.travis.yml` for you. Note that the `region` isn't generated by running `travis setup opsworks`.
```
Comment (plaindocs) [inline user/deployment/opsworks.md]: Nice idea to fold this into the note about `setup`.

File: user/deployment/opsworks.md [hunk 2]
```diff
@@ -14,9 +14,12 @@ deploy:
   access-key-id: ACCESS-KEY-ID
   secret-access-key: SECRET-ACCESS-KEY
   app-id: APP-ID
+  region: REGION # (default: us-east-1)
 ```
 {: data-file=".travis.yml"}
 
+Strictly, `region` isn't required but it defaults to `us-east-1` so you'll need it if your stack is elsewhere. You will get an error like "OpsWorks service error: Unable to find app with ID xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx" if your stack is not in `us-east-1` and you don't define `region`.
```
Comment (plaindocs) [inline user/deployment/opsworks.md]: This is now a little repetitive, how about:

>  region` isn't required but it defaults to `us-east-1`. If your application is located in a different region you will get an error like "OpsWorks service error: Unable to find app with ID xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx".

Or do you think we need to be more clear?

Comment (billhorsman) [reply user/deployment/opsworks.md]: Nice. That reads much better. I've incorporated your suggestion :)

**Comment by plaindocs** [comment]
Thanks again!

**Pull request merged**